Editor/UI/EditorWindows/EditorWindowBase.cs

```diff
@@ -54,34 +59,33 @@ private void LoadAssets()
                 }
             };
 
-            if (DescriptionStyle == null)
+            DescriptionStyle ??= new GUIStyle
             {
-                DescriptionStyle = new GUIStyle
+                fontSize = 12,
+                richText = true,
+                wordWrap = true,
+                margin = new RectOffset(5, 0, 0, 0),
+                normal =
                 {
-                    fontSize = 12,
-                    richText = true,
-                    wordWrap = true,
-                    margin = new RectOffset(5, 0, 0, 0)
-                };
-                DescriptionStyle.normal.textColor = new Color(0.7f, 0.7f, 0.7f, 1.0f);
-            }
+                    textColor = new Color(0.7f, 0.7f, 0.7f, 1.0f)
+                }
+            };
 
-            if (webButtonStyle == null)
+            webButtonStyle ??= new GUIStyle(GUI.skin.button)
             {
-                webButtonStyle = new GUIStyle(GUI.skin.button);
-                webButtonStyle.fontSize = 12;
-                webButtonStyle.fixedWidth = 149;
-                webButtonStyle.fixedHeight = ButtonHeight;
-                webButtonStyle.padding = new RectOffset(5, 5, 5, 5);
-            }
+                fontSize = 12,
```

Comment:
minor thing but we have font size hardcoded to 12 in two places, we could instead store it in a constant.